inventory/inventory

```diff
@@ -12,5 +12,8 @@
 10.0.1.2
 10.0.1.3
 
+[centos7:vars]
+ansible_password=student
```

Comment:
Normally [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) is used to manage secrets like this. To make lab submissions easier though, it's okay to leave it public as-is. Maybe something to mention in the sec report?

Reply:
I guess, but even then, the playbook appears frozen after I run it

![image](https://user-images.githubusercontent.com/37078653/64928684-b8d79a00-d7e9-11e9-857a-6e28cc5d33e9.png)

been sitting like that for the past 30 minutes or so

Reply:
@BonjeBongo Weird. Try running with more verbosity to get better explanations of what is going on underneath: `ansible-playbook -vv playbooks/master.yml`

Reply:
Did that, after the ok for 10.0.1.3 its just blank. And just sits there. Check the Discord